src/api/apicall.js

```diff
@@ -0,0 +1,85 @@
+import axios from "axios";
+const auth =
+	"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJfaWQiOiI4MTAyMWNjNC00YjFkLTQyOGItYjJmMC0wNjhkYTQ4YTk4MzQiLCJlbWFpbCI6ImFkYXJzaGJhbGlrYUBnbWFpbC5jb20ifQ.45ynQ6aZhoM1zsNwIKCYR_IATaszKn0ssvnPPQkKL8E";
+const getAllProducts = async () => {
+	try {
+		const response = await axios.get("/api/products");
+		return { products: response.data.products, success: true };
+	} catch (err) {
+		console.log(err);
+		return { products: [], success: false };
+	}
+};
+
+const getWishlist = async () => {
+	const headers = { authorization: auth };
```

Comment:
Since headers is used in all the functions you can declare it once after auth itself instead of declaring it every time inside a function

Reply:
Fixed the price slider issue.
removed headers repetition.